src/sonic-config-engine/minigraph.py

```diff
@@ -1174,6 +1180,13 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         }
     }
 
+    if kube_data:
+        results['KUBERNETES_MASTER'] = { 'SERVER': {
+            'disable': 'True' if kube_data.get('disable', '0') == '0' else 'False',
```

Comment:
>'True' [](start = 23, length = 6)

If you need True/False string, you can use str(bool)

Reply:
Yep, a better one. Taken.